frontend/lib/app.dart

```diff
@@ -104,29 +105,31 @@ class App extends StatelessWidget {
     });
 
     return Configuration(
-      mapStyleUrl: mapStyleUrl,
-      graphqlUrl: graphqlUrl,
-      projectId: projectId,
-      showDevSettings: kDebugMode,
-      child: ConfiguredGraphQlProvider(
-        child: ChangeNotifierProvider(
-          create: (context) => UserCodeModel()..initialize(),
-          child: MaterialApp.router(
-            theme: lightTheme,
-            darkTheme: darkTheme,
-            themeMode: ThemeMode.system,
-            debugShowCheckedModeBanner: false,
-            localizationsDelegates: [
-              GlobalMaterialLocalizations.delegate,
-              GlobalWidgetsLocalizations.delegate,
-              GlobalCupertinoLocalizations.delegate,
+        mapStyleUrl: mapStyleUrl,
```

Comment:
This lines did not change, but are displayed here as "changed", due to formatting changes. Can you please undo. Not sure why the linter did not catch this.